website/pages/chapters/ridge.js

```diff
@@ -0,0 +1,845 @@
+              problem in machine learning applications. In an ML task, we are
+              specifically looking for correlations in the data and
+              extrapolating from them.
+            </p>
+            <p>
+              This is how you get awful machine learning systems used in the
+              criminal justice system that predict that black people are more
+              likely to recommit a crime than their white counterparts
+              <MarginNote>
+                <a href="https://www.propublica.org/article/machine-bias-risk-assessments-in-criminal-sentencing">
+                  Machine Bias - ProPublica
+                </a>
+                .
+              </MarginNote>
+              . The model can't tell the difference between a correlation and a
+              causation. A very clear underlying factor that can explain this
+              correlation is the historical and current anti-black bias in the
+              criminal justice system: Black communities are more likely to be
+              policed and are more likely to be sentenced for minor crimes than
+              their white counterparts. We discuss this more in the chapter on
+              fairness and ML.
+            </p>
+          </li>
+        </ul>
+
+        <p>
+          We will talk more about interpretability throughout the book. While it
+          is a useful aspect of some models we use, it is important to remember
+          the context that we are using them in and be careful in our analysis.
+        </p>
+      </section>
+
+      <section>
+        <h2>Overfitting</h2>
+        <p>
+          Consider our housing example, where we try to predict the price of its
+          house from the square footage. In earlier chapters, we said we could
+          use polynomial regression with degree <IM math="p" />. As a reminder,
+          the model we use there would predict{" "}
+          <IM math={`\\hat{y} = \\sum_{j=0}^p w_jx^p`} />.
+        </p>
+
+        <p>TODO(picture): High degree polynomial through data</p>
+
+        <p>
+          In the previous chapter, we said this model is likely to overfit if it
+          is too complex for the data you have. While we defined overfitting in
+          terms of the training error and true error, there is another way to
+          identify if your model might be overfit. Polynomial regression models
+          tend to have really large coefficients as they become more and more
+          overfit. In other words, in order to get that really wiggly behavior
+          in the polynomial, the coefficients must be large in magnitude (
+          <IM math={`|\\hat{w}_j| >> 0`} />
+          ). The following animation shows how increasing the degree{" "}
+          <IM math="p" /> of a polynomial regression model tends to increase the
+          magnitudes of the coefficients.
+        </p>
+
+        <p>TODO(manim) coefficient growth?</p>
```

Comment:
Sorry, I don't think I quite follow what you mean here. I think we established why these wiggly polynomials are bad in the Assessing Performance chapter so I'm not sure if we need another animation to show that it's bad again. 

I should have maybe written a clearer note for what I meant here since this is extremely vague. For this animation, I was intending to show that as you increase the degree `p`, the magnitudes of the coefficients increase. There is a plot of this in that Jupyter notebook I made where you can see that the coefficients get HUGE even for moderate `p`.

Reply:
Coming back I don't really know what i was trying to get at either.